lms/djangoapps/teams/static/teams/js/spec/views/edit_team_spec.js

```diff
@@ -173,7 +173,7 @@ define([
             AjaxHelpers.respondWithError(
                     requests,
                     400,
-                    {'error_message': {'user_message': 'User message', 'developer_message': 'Developer message' }}
+                    {'user_message': 'User message', 'developer_message': 'Developer message'}
```

Comment:
Note to reviewers: simplified the error message by not nesting the user_message/developer_message fields within another dict (to be consistent with the rest of our API).